### ISSUE TITLE
Replacing a letter in a url

### ISSUE THREAD

**Issue by clement-moulin-frier** [daverix/urlforwarder#13]
Hi,

Thanks for the app. I'd like to use it for the following use case by I can't figure out how to do it:

When browsing the url `www.example.com/t`, I would like to automatically redirect to `www.example.com/m`. 
(I.e, I just want to replace the `t` at the end of the url by a `m`.)

I have tried many different configurations but none of them worked. Could you help me on this?

Thanks a lot!

**Comment by daverix** [comment]
Hi, it doesn't work like that, you need to share the current page to be redirected to the new page. Simply browsing the page is not supported.

**Comment by clement-moulin-frier** [comment]
Yes I understood that I need to share the webpage with urlforwarder.
If I understand well what you say in your second paragraph, I have to configure it with (using my example use case described in my first post):
 Filter url: `www.example.com/t/`
 Replaceable text: `www.example.com/m/`

I have tried it but it redirects me to the original url `www.example.com/t/` instead of the desired one `www.example.com/m/`.

Thanks!

**Comment by daverix** [comment]
Okay now I understand what you mean.

It's not possible currently to remove anything from the url you want to forward to.
